packages/design-tokens/stories/test-utils/getCSSVariablesFromStylesheet.ts

```diff
@@ -1,7 +1,10 @@
-import fs from 'fs';
-import path from 'path';
-import postcss from 'postcss';
+/* eslint-disable no-restricted-globals */
+/* eslint-disable id-denylist */
+/* eslint-disable prefer-destructuring */
+/* eslint-disable @typescript-eslint/prefer-for-of */
+/* eslint-disable require-unicode-regexp */
```

Comment:
Need to disable linting for this file as it uses forbidden globals to collect the stylesheet. We can't use the stylesheets from design-tokens because it conflicts with the node environment